src/libpisp/backend/tiling/crop_stage.cpp

```diff
@@ -72,3 +97,8 @@ void CropStage::PushCropDown(Interval interval, Dir dir)
 	PISP_LOG(debug, "(" << name_ << ") Exit with interval " << output_interval_);
 	downstream_->PushCropDown(output_interval_, dir);
 }
+
+void CropStage::Reset()
+{
```

Comment:
Should we call the parent `BasicStage::Reset()`? I'm guessing it makes no difference...